### PR TITLE
add Chinese (Simple) 

### PR DESCRIPTION
**Pull request by qa003qa003**
add Chinese (Simple)

### CODE REVIEW

**Review by DerDakon — CHANGES_REQUESTED** [review]
Please drop the commit that introduces locals/messages.po as this is just a duplicate. Also please add an entry to api/config.php so it will be selectable. At the end please squash everything together in a single commit.

File: locales/zh_CN/LC_MESSAGES/messages.po [hunk 1]
```diff
@@ -0,0 +1,778 @@
...
+msgstr ""
+
+msgid "Punktförmige Zugbeeinflussung"
+msgstr ""
+
+msgid "Operating Sites"
+msgstr ""
+
+msgid "train protection"
+msgstr ""
```
Comment (myzhang1029) [inline locales/zh_CN/LC_MESSAGES/messages.po:667]: `"列车保护系统"`.

File: locales/zh_CN/LC_MESSAGES/messages.po [hunk 2]
```diff
@@ -0,0 +1,778 @@
...
+
+msgid "contact line"
+msgstr ""
```
Comment (myzhang1029) [inline locales/zh_CN/LC_MESSAGES/messages.po:670]: `"接触网"`.

File: locales/zh_CN/LC_MESSAGES/messages.po [hunk 3]
```diff
@@ -0,0 +1,778 @@
...
+
+msgid "15 – 25kV ~"
+msgstr ""
+
+msgid "ETCS"
+msgstr ""
+
+msgid "Electrification under construction"
+msgstr ""
```
Comment (myzhang1029) [inline locales/zh_CN/LC_MESSAGES/messages.po:577]: `"电气化在建"`.

File: locales/zh_CN/LC_MESSAGES/messages.po [hunk 4]
```diff
@@ -0,0 +1,778 @@
...
+
+msgid "At least 25kV ~"
+msgstr ""
+
+msgid "minor signal type Lo"
+msgstr ""
+
+msgid "Finland"
+msgstr ""
```
Comment (myzhang1029) [inline locales/zh_CN/LC_MESSAGES/messages.po:568]: `"芬兰"`.

**Review by myzhang1029 — CHANGES_REQUESTED** [review]
I've proofread most of the English-to-SC translations, and I kindly suggest some changes to be considered.

File: locales/zh_CN/LC_MESSAGES/messages.po [hunk 5]
```diff
@@ -0,0 +1,778 @@
...
+msgstr[0] "％d个月"
+msgstr[1] "％d个月"
+
+"#, php-format"
+msgid "more than %s ago"
+msgstr "％s之前"
+
+"#, php-format"
+msgid "%s ago"
+msgstr "％s前"
```
[the remark this answers is not in the document]
Comment (DerDakon) [reply locales/zh_CN/LC_MESSAGES/messages.po:142]: Yes, absolutely. Nice catch, I didn't spot this.

**Comment by DerDakon** [comment]
Fixes: #735

File: locales/zh_CN/LC_MESSAGES/messages.po (hunk 5 above)
[the remark this answers is not in the document]
Comment (Nakaner) [reply locales/zh_CN/LC_MESSAGES/messages.po:142]: Yes, the `%` is required for string formatting.

**Comment by Nakaner** [comment]
Closed in favour of #740

**Pull request closed without merging**